### PR TITLE
Check spelling of forward in config

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/crossbar/priv/couchdb/schemas/system_config.callflow.json [hunk 1]
```diff
@@ -150,9 +150,13 @@
                     "description": "callflow save after notify",
                     "type": "boolean"
                 },
-                "vm_message_foraward_type": {
+                "vm_message_forward_type": {
                     "default": "only_forward",
-                    "description": "Enable or disable the ability to prepend a message when forwarding a voicemail message",
+                    "description": "callflow voicemail vm_message_forward_type",
```
Comment (icehess) [inline applications/crossbar/priv/couchdb/schemas/system_config.callflow.json]: Why changing the description?

Comment (jamesaimonetti) [reply applications/crossbar/priv/couchdb/schemas/system_config.callflow.json]: Oops, I didn't realize the old one had a "proper" description :)

**Comment by jamesaimonetti** [comment]
My guess, re migration, is that no one has used this setting yet; otherwise they would have noticed the spelling. :) But probably better safe than sorry; I'll add migration.

**Comment by jamesaimonetti** [comment]
@icehess have your migration!

**Pull request merged**